open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/authservlet/impl/OBDefaultAuthServletImpl.java

```diff
@@ -91,8 +93,11 @@ public String getJSPPath() {
 
         if (jspPath.equalsIgnoreCase(ConsentExtensionConstants.ACCOUNTS)) {
             return "/ob_default.jsp";
+        } else if (jspPath.equalsIgnoreCase(ConsentExtensionConstants.VRP)) {
```

Comment:
Can't we include both account and vrp in same if statement

Reply:
Fixed in [a6d2b0b]